nylium.Core/World/World.cs

```diff
@@ -77,18 +79,11 @@ public class World {
 
         // TODO better way to do this?
         public Chunk GetChunk(int x, int z) {
-            for(int i = 0; i < Chunks.Count; i++) {
-                Chunk chunk = Chunks[i];
-
-                if(chunk.X == x && chunk.Z == z) {
-                    return chunk;
-                }
+            try {
```

Comment:
It would be better to check whether the entry exists or not with .ContainsKey()

Reply:
why

Reply:
this is fine